packetbeat/protos/tls/parse.go

```diff
@@ -639,7 +639,7 @@ func certToMap(cert *x509.Certificate) mapstr.M {
 	certMap := mapstr.M{
 		"signature_algorithm":  cert.SignatureAlgorithm.String(),
 		"public_key_algorithm": toString(cert.PublicKeyAlgorithm),
-		"serial_number":        cert.SerialNumber.Text(10),
+		"serial_number":        strings.ToUpper(cert.SerialNumber.Text(16)),
```

Comment:
I think it should be prefixed with `0x`, because I expect hex numbers to be that way. But, whatever.

Reply:
same, but the ECS docs seem to imply it shouldn't have the `0x`?

```
Unique serial number issued by the certificate authority. For consistency, if this value is alphanumeric, it should be formatted without colons and uppercase characters.

type: keyword

example: 55FBB9C7DEBF09809D12CCAA
```

Reply:
Following the example makes sense to me.